packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -142,6 +147,14 @@ pub struct PersistedPropertyType {
     pub identifier: PersistedOntologyIdentifier,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Component)]
+#[serde(rename_all = "camelCase")]
+pub struct PropertyTypeTree {
+    pub property_type: PersistedPropertyType,
+    pub data_type_references: Vec<PersistedDataType>,
```

Comment:
Can we actually rename all of these to `referenced_data_types` and such? We're not returning references

Reply:
Good point: cad22e2abd3235f1c610e6cafadc1035a2fef216, this will also prettify the usage at the backend